src/main/java/frc5124/robot2020/subsystems/DriveTrain.java

```diff
@@ -50,6 +52,11 @@ public DriveTrain() {
         rightFollower = new WPI_TalonFX(RobotMap.DriveTrainMap.rightFollowerCanID);
         rightFollower.follow(rightLeader);
 
+        rightLeader.setNeutralMode(NeutralMode.Brake);
+        rightFollower.setNeutralMode(NeutralMode.Brake);
+        leftLeader.setNeutralMode(NeutralMode.Brake);
+        leftFollower.setNeutralMode(NeutralMode.Brake);
+
```

Comment:
Does Will no longer want it to be half and half, or is this just the wrong code version?

Reply:
wrong ver